### PR TITLE
✅ Validate <amp-video-iframe>

### PR DESCRIPTION
**Pull request by alanorozco**

### CODE REVIEW

File: extensions/amp-video-iframe/validator-amp-video-iframe.protoascii [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+  }
+  attrs: {
+    name: "implements-media-session"
+    value: ""
+  }
+  attrs: {
+    name: "implements-rotate-to-fullscreen"
+    value: ""
+  }
+  attrs: {
```
Comment (aghassemi) [inline extensions/amp-video-iframe/validator-amp-video-iframe.protoascii]: `preload` this doesn't apply here

File: extensions/amp-video-iframe/validator-amp-video-iframe.protoascii [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+  extension_spec: {
+    name: "amp-video-iframe"
+    version: "0.1"
+    version: "latest"
+  }
+  attr_lists: "common-extension-attrs"
+}
+
+attr_lists: {
+  name: "amp-video-iframe-common"
```
Comment (aghassemi) [inline extensions/amp-video-iframe/validator-amp-video-iframe.protoascii]: let's add `referrerpolicy` and make sure runtime propagates it.

File: extensions/amp-video-iframe/validator-amp-video-iframe.protoascii [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+    value: "auto"
+    value: "metadata"
+    value: "none"
+    value: ""
+  }
+  attrs: {
+    name: "rotate-to-fullscreen"
+    value: ""
+  }
+  attrs: {
```
Comment (aghassemi) [inline extensions/amp-video-iframe/validator-amp-video-iframe.protoascii]: wanna do `dock` for this now or later? (I think `dock` should launch with amp-video, amp-video-iframe and amp-youtube support as MVP)

**Comment by alanorozco** [comment]
@honeybadgerdontcare Sorry for pulling the trigger too soon D:

Let me know if you'd like any changes to be made, since this has not been released yet.

**Comment by honeybadgerdontcare** [comment]
@alanorozco 	Thanks Alan! Per offline discussion for the other amp-video-iframe items we'll have to make some changes to the engine.

**Comment by AmyLiversidge** [comment]
@alanorozco @honeybadgerdontcare Is there is any update this? We're really keen to start using this component.

**Comment by alanorozco** [comment]
@AmyLiversidge Hi, this component will be out in production the first week of January next year. It's currently valid (barring pending validator updates) and feature-complete, but it's guarded behind an experiment flag. This guard will be removed on said week.

**Comment by AmyLiversidge** [comment]
Hi @alanorozco, please could you give me an update on this? Many thanks.

**Comment by alanorozco** [comment]
Hi @AmyLiversidge,

This has been released without experiment guards on the dev channel and should be available in the next production release.

**Pull request merged**